src/AutoRest.CSharp/Common/Input/InputTypes/InputLiteralType.cs

```diff
@@ -3,4 +3,7 @@
 
 namespace AutoRest.CSharp.Common.Input;
 
-internal record InputLiteralType(string Name, InputType LiteralValueType, object Value, bool IsNullable) : InputType(Name, IsNullable);
+internal record InputLiteralType(InputConstant Value, bool IsNullable) : InputType(InputLiteralTypeName, IsNullable)
```

Comment:
We may not update the Name here, we need to reflect everything from the emitter.